spec/features/user_edits_profile_spec.rb

```diff
@@ -16,5 +16,15 @@
 
     expect(page).to have_content("Profile Page")
     expect(page).to have_content("Sign Out")
+
+    fill_in 'First name', with: 'Test'
+    fill_in 'Last name', with: 'Update'
+    fill_in 'Address', with: '10 Elm St'
+    fill_in 'City', with: 'Boston'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.